homeassistant/components/tellstick/__init__.py

```diff
@@ -179,7 +179,10 @@ def __init__(self, tellcore_device, signal_repetitions):
 
         # Look up our corresponding tellcore device
         self._tellcore_device = tellcore_device
-        self._name = tellcore_device.name
+        self._attr_assumed_state = True
+        self._attr_name = tellcore_device.name
+        self._attr_should_poll = False
```

Comment:
`should_poll` and `assumed_state` should be static class attributes, not instance attributes:
```python
    _attr_assumed_state = True
    _attr_should_poll = False
```

Reply:
They are already instances from the parent Entity object.

Reply:
@michaelarnauts  He means it like this:

```py
class TellstickDevice(Entity):
    """Representation of a Tellstick device.
    Contains the common logic for all Tellstick devices.
    """

    _attr_assumed_state = True
    _attr_should_poll = False
```

Reply:
Oh, sure! I'll change this. 👍